crates/matrix-sdk/src/room/mod.rs

```diff
@@ -380,7 +381,13 @@ impl Room {
             .members_request_deduplicated_handler
             .run(self.room_id().to_owned(), async move {
                 let request = get_member_events::v3::Request::new(self.inner.room_id().to_owned());
-                let response = self.client.send(request, None).await?;
+                let response = self
+                    .client
+                    .send(
+                        request,
+                        Some(RequestConfig::new().timeout(Duration::from_secs(60)).retry_limit(3)),
```

Comment:
Can you add a comment that for very large rooms, Synapse returns the response in more than 30 seconds, and that's why we need to bump this timeout value?

Reply:
(Matrix SDK should not mention `Synapse`, using generic terms like `the Matrix server` is better I think)